runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -598,4 +505,136 @@ private Layout getHighestEpochLayout(Map<String, Layout> layoutMap) {
                     return true;
                 });
     }
+
+    public static class ClusterHealth {
```

Comment:
Where these functions just moved, or was there any change in logic?

Reply:
Just moved from ManagementView to ClusterHealth